acceptance/tests/environment/environment_scenario-bad.rb

```diff
@@ -16,32 +16,32 @@
 apply_manifest_on(master, environment_manifest(testdir), :catch_failures => true)
```

Comment:
@camlow325 i guess i have not been able to reproduce your issue.  using the code as is, if i add agents to our masters the tests don't fail.